subprojects/test-runner/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/internal/report/listener/ReportProcessorImpl.kt

```diff
@@ -153,7 +156,8 @@ internal class ReportProcessorImpl(
                         incident = Incident(
                             type = Incident.Type.INFRASTRUCTURE_ERROR,
                             timestamp = now,
-                            trace = problem.asRuntimeException().stackTraceToList(),
+                            trace = problem.throwable?.stackTraceToList()
+                                ?: problem.asRuntimeException().stackTraceToList(),
```

Comment:
@sboishtyan that would act as before, i introduced another layer of exception on top of original cause, which is kinda useless;